test/worlds/model_nested_include.sdf

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" ?>
+<sdf version="1.8">
+  <world name="model_nested_include_world">
+    <physics name="1ms" type="ignored">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="libignition-gazebo-physics-system.so"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <plugin
+      filename="libignition-gazebo-scene-broadcaster-system.so"
+      name="ignition::gazebo::systems::SceneBroadcaster">
+    </plugin>
+
+    <scene>
+      <ambient>1.0 1.0 1.0</ambient>
+      <background>0.8 0.8 0.8</background>
+    </scene>
+
+    <model name="ground_plane">
+      <static>true</static>
+      <link name="L0">
+        <collision name="C0">
+          <geometry>
+            <box>
+              <size>20 20 0.1</size>
+            </box>
+          </geometry>
+        </collision>
+        <visual name="V0">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+          <material>
+            <ambient>0.8 0.8 0.8 1</ambient>
+            <diffuse>0.8 0.8 0.8 1</diffuse>
+            <specular>0.8 0.8 0.8 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+
+    <model name="M1">
+      <include>
+        <uri>include_nested</uri>
+        <name>include_nested_new_name</name>
+        <pose>1 2 3 0 0 0</pose>
+      </include>
+
+      <include>
```

Comment:
Can you include a Fuel model here?

Reply:
1ff35d0c9032f60d5a74fb6b803944230f318ca5